RELEASE.md

```diff
@@ -0,0 +1,15 @@
+# Releases
+This page describes the release process and the currently planned schedule for upcoming releases as well as the respective release shepherd. Release shepherds are chosen on a voluntary basis.
+
+## Release Schedule
+
+Release cadence of first pre-releases being cut is 12 weeks.
+
+| release series | date (year-month-day) | release shepherd              |
+|----------------|-----------------------|-------------------------------|
+| v0.26          | 2023-05-31            | Josh Abreu (Github: @gotjosh) |
+| v0.27          | 2023-08-31            | Josh Abreu (Github: @gotjosh) |
```

Comment:
you can sign me up for this one if you don't want to do twice in a row

Reply:
I'd be happy to do it twice in a row - you can take the third.

Reply:
:+1: